src/edit_collector.h

```diff
@@ -121,20 +128,33 @@ class EditCollector {
  private:
   class CFEditCollector {
    public:
+    CFEditCollector(Logger* logger, bool _paranoid_check)
+        : paranoid_check_(_paranoid_check), info_log_(logger) {}
+
     Status AddFile(const std::shared_ptr<BlobFileMeta>& file) {
       auto number = file->file_number();
       if (added_files_.count(number) > 0) {
-        return Status::Corruption("Blob file " + ToString(number) +
-                                  " has been added twice");
+        TITAN_LOG_ERROR(info_log_,
+                        "blob file %" PRIu64 " has been deleted twice\n",
+                        number);
+        if (paranoid_check_) {
+          return Status::Corruption("Blob file " + ToString(number) +
```

Comment:
If paranoid_check_ is false, I think it can return Status::OK immediately, as emplace will not overwrite existing key.